src/main/java/org/takes/rq/RqFake.java

```diff
@@ -133,4 +122,39 @@ public InputStream body() {
         );
     }
 
+    /**
+     * The request which is used when the body is a byte array.
+     */
+    @EqualsAndHashCode
+    static final class RqFakeRequest implements Request {
```

Comment:
@bdragan instead of a new class, why not make `RqFake` have these fields and just implement `Request` instead of extending `RqWrap`?